src/Components/Moormanage/Technicians/Technicians.tsx

```diff
@@ -0,0 +1,200 @@
+import { useEffect, useState } from 'react'
+import { DataTable } from 'primereact/datatable'
+import { Column } from 'primereact/column'
+import { SelectButton, SelectButtonChangeEvent } from 'primereact/selectbutton'
+import { Nullable } from 'primereact/ts-helpers'
+import { Button } from 'primereact/button'
+import { Calendar } from 'primereact/calendar'
+import { TechnicianPayload, TechnicianResponse } from '../../../Type/ApiTypes'
+import { useGetTechnicianMutation } from '../../../Services/MoorManage/MoormanageApi'
+import { BillsData } from '../../../Type/CommonType'
+
+const Technicians = () => {
+  const [date, setDate] = useState<Nullable<(Date | null)[]>>(null)
+  const options: string[] = ['Open', 'Completed']
+  const [value, setValue] = useState<string>(options[0])
+  const [dataVisible, setDataVisible] = useState(false)
+  const [technicianRecord, setTechnicianRecord] = useState()
+  const [globalFilter, setGlobalFilter] = useState<string | null>(null)
```

Comment:
instead of null change to undefined

Reply:
it is resolved ?

Reply:
yes i chnged the null to undefined

Reply:
![Uploading Screenshot from 2024-04-29 14-55-05.png…]()